index.html

```diff
@@ -273,5 +274,49 @@ <h3>Skills</h3>
         <hr />
       </ul>
     </div>
+    <footer class="contact">
+      <section class="background2">
+        <form
+          class="form"
+          action="https://formspree.io/f/xdovrwzl"
+          method="POST"
+        >
+          <div class="section3">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it!
+            </p>
+            <input
+              id="name"
+              class="name"
+              name="name"
+              type="text"
+              placeholder="Lorem Ipsum"
+              maxlength="30"
+            />
+            <input
+              id="email"
+              class="email"
+              name="email"
+              type="email"
+              placeholder="loremipsum@gmail.com"
+            />
+            <textarea
+              id="text"
+              class="text"
+              type="text"
+              name="message"
+              placeholder="Write your message here..."
+              maxlength="500"
+            ></textarea>
+          </div>
+          <button type="submit" value="submit" class="submit">
+            Get in touch
+          </button>
+        </form>
+      </section>
+    </footer>
```

Comment:
Consider using Flexbox to arrange and position your elements. Flexbox is a powerful CSS layout module that provides a flexible and responsive way to organize content. It allows you to create dynamic and adaptable layouts, especially for responsive web design.

Reply:
This issue has been resolved.